src/main/java/at/htl/persistence/entity/Unit.java

```diff
@@ -0,0 +1,77 @@
+package at.htl.persistence.entity;
+
+import at.htl.persistence.dao.LessonDao;
+import at.htl.rest.dto.RoomDto;
+import at.htl.rest.dto.UnitDto;
+
+import javax.persistence.*;
+import java.sql.Timestamp;
+import java.util.List;
+
+@Entity
+public class Unit {
+    @Id
+    @SequenceGenerator(sequenceName = "Unit_Seq", name="Unit_Seq")
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "Unit_Seq")
+    private Integer id;
+    private Timestamp startTime;
+    private Timestamp endTime;
+    @ManyToMany
+    private List<Lesson> lessons;
+
+    //region Constructors
+    public Unit() {
+    }
+
+    public Unit(Integer unitId, Timestamp startTime, Timestamp endTime) {
+        this.id = unitId;
+        this.startTime = startTime;
+        this.endTime = endTime;
+    }
+    //endregion
+    public UnitDto toDto(){
+        UnitDto dto = new UnitDto();
+        dto.setId(id);
```

Comment:
I don't think that we need to check whether timestamp is null, since we want the timestamp null anyway if the timestamp is missing